commands2/sysid/sysidroutine.py

```diff
@@ -96,8 +96,8 @@ def __init__(self, config: Config, mechanism: Mechanism):
         self.config = config
         self.mechanism = mechanism
         self.outputVolts = 0.0
-        if config.recordState != None:
-            self.recordState = config.recordState
+        if config.logState != None:
+            self.record_State = config.logState
```

Comment:
Don't you want this new member variable to be called below?

Reply:
I am not sure I understand. Is it not still being called in the generated commands since I modified the name of the input config?

Reply:
This callable is being stored, but isn't used anywhere. Shouldn't it be called in the same places it's called in the C++ and Java versions?